src/applications/disability-benefits/686c-674/config/chapters/veteran-information/veteran-information/VeteranInformationComponent.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import AlertBox from '@department-of-veterans-affairs/formation-react/AlertBox';
+import { connect } from 'react-redux';
+import moment from 'moment';
+
+const VeteranInformationComponent = ({
+  user: {
+    gender,
+    dob,
+    userFullName: { first, last },
+  },
+}) => {
+  let dateOfBirthFormatted = '-';
+  let genderFull = '-';
+  if (dob) {
+    dateOfBirthFormatted = moment(dob).format('MMMM Do YYYY');
+  }
+  if (gender === 'M') {
+    genderFull = 'Male';
+  } else if (gender === 'F') {
+    genderFull = 'Female';
+  }
```

Comment:
This isn't crucial, but you could use some constants that are already defined in the platform for this if you want:

https://github.com/department-of-veterans-affairs/vets-website/blob/b81737f159ed563a0d52edc5ce54d063b6e3acc0/src/platform/static-data/labels.jsx#L6-L10